pom.xml

```diff
@@ -68,6 +68,7 @@
     <ver.commonslang3>3.4</ver.commonslang3>
     <ver.commonscsv>1.5</ver.commonscsv>
     <ver.commons-codec>1.11</ver.commons-codec>
+    <ver.commons-compress>1.16.1</ver.commons-compress>
```

Comment:
1.17 was just released... maybe worth using it instead? Just received Stefan's announcement message about it in the commons mailing list.

Reply:
Yes! Thanks for the pointer.